sdk/tests/conformance/rendering/rendering-sampling-feedback-loop.html

```diff
@@ -125,10 +134,12 @@
     }
 }
 
-function init() {
+function init(drawBuffers) {
```

Comment:
Oops I forgot to submit my other comment. drawBuffers can be captured by scope without passing it as an argument. Since rendering_sampling_feedback_loop captures by scope, let's be consistent and do it here too. (remove the drawBuffers arg)